docs/runtime-configuration.md

```diff
@@ -0,0 +1,349 @@
+# Runtime configuration
+
+The apu platforms firmware contain an application called sortbootorder which
+serves as a BIOS setup application. It allows to modify behavior of the
+coreboot firmware and enable or disable features/peripherals.
+
+For more information about the usage of sortbootorder application refer to
+sortbootorder [README](https://github.com/pcengines/sortbootorder/blob/master/README.md).
+
+## Offline runtime configuration
+
+An application to modify the runtime configuration of apu platforms has been
+developed. The source code is available at [PC Engines GitHub repository](https://github.com/pcengines/cb-order).
+
+This application allows modifications of runtime configuration options on a
+coreboot binary without the need of rebooting the platform and entering setup
+menu via BIOS (which requires serial console). The application offers either a
+user-friendly GUI or command line interface for scripting and automation.
+Please refer to the [application's README](https://github.com/pcengines/cb-order/blob/master/README.md)
+for the building process and dependencies.
+
+For the usage you may execute `cb-order -h`.
+
+Example command line usage:
+`cb-order coreboot.rom -b iPXE,SATA -o usben=off -o watchdog=300`
+
+This command will set the boot order to be 1st iPXE then 2nd SATA. Additionally
+it disables booting from USB and sets the watchdog to 300 seconds.
```

Comment:
I'd rephrase it a bit to suggest that this is a reordering and other boot options are still there:
```suggestion
This command moves iPXE and SATA boot options to the 1st and 2nd positions
respectively displacing other options from those positions. Additionally it
disables booting from USB and sets the watchdog to 300 seconds.
```